src/main/java/net/imagej/legacy/plugin/MacroAutoCompletionProvider.java

```diff
@@ -153,16 +153,19 @@ void addModuleCompletions(ModuleService moduleService) {
 		sorted = false;
 		this.moduleService = moduleService;
 
+		ArrayList<Completion> list = new ArrayList<Completion>();
```

Comment:
Can we name this more concisely `completions`, as in the `addMacroExtensionAutoCompletions` method? Also, the redundant type arguments can be removed from the constructor call:

```suggestion
		ArrayList<Completion> completions = new ArrayList<>();
```